composer.json

```diff
@@ -33,14 +33,14 @@
     ],
     "require": {
         "composer/installers": "^1.7",
-        "consolidation/robo": "^3.0",
         "cweagans/composer-patches": "^1.7",
         "drupal-composer/drupal-security-advisories": "8.x-dev",
         "drupal/admin_toolbar": "^3.3",
-        "drupal/components": "^2.4",
+        "drupal/components": "^3.0@beta",
         "drupal/config_split": "^2.0@RC",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-composer-scaffold": "^10.0",
+        "drupal/core-recommended": "^10.0",
+        "drupal/drupal-extension": "^5.0@alpha",
```

Comment:
- Drupal extension is  a dev library and it is being not required as require-dev
- Drupal extension is not required by required by the boilerplate but by nuvoleweb/drupal-behat which is already required by metadrop/behat-contexts, we should not require it as we rely on metadrop/behat-contexts to require the complete stack.

```suggestion
```